src/modules/shared/components/Hero/Hero.module.scss

```diff
@@ -1,6 +1,9 @@
 @use "sass:map";
 @use "styles/abstracts" as *;
 
+$c-hero-logged-in-bg: #F0F0F0;
```

Comment:
Ik zou dit naar `$platinum` veranderen, lijkt een design inconsistentie want die kleur is niet benoemd in XD.